hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/fees/FeesAndRatesProvider.java

```diff
@@ -190,7 +189,7 @@ private FileGetContentsResponse downloadWith(long queryFee, boolean costOnly, Fi
                     .getFileContent(query)
                     .getFileGetContents();
             status = response.getHeader().getNodeTransactionPrecheckCode();
-            if (status == OK || status == PLATFORM_NOT_ACTIVE) {
```

Comment:
Why this change was needed? Looks like @jsync-swirlds added this check(https://github.com/hashgraph/hedera-services/pull/9197/files#diff-9bbb90112981633f33079ee75e18634d4dbab99a634c63baa6a119b17a892117R193) two days ago.

Reply:
@jsync-swirlds has his pr 9212 to remove this as well and already merged. I will merge from develop.